pub_sub/python/sdk/README.md

```diff
@@ -22,7 +22,8 @@ name: run
 
 ```bash
 cd ./order-processor
-pip3 install -r requirements.txt 
+pip3 install -r requirements.txt
+pip3 install typing-extensions
```

Comment:
`typing-extensions` needs to be a new line in requirements.txt, and not a new command the user needs to run